frappe/desk/search.py

```diff
@@ -272,3 +306,12 @@ def get_user_groups():
 	return frappe.get_all('User Group', fields=['name as id', 'name as value'], update={
 		'is_group': True
 	})
+
+@frappe.whitelist()
+def get_link_title(doctype, docname):
+	meta = frappe.get_meta(doctype)
+
+	if meta.title_field and meta.show_title_field_in_link:
+		return frappe.get_cached_value(doctype, docname, meta.title_field)
```

Comment:
Should I change this to `frappe.db.get_value`, since this internally calls `frappe.get_doc` if the doc is not found in cache